app/components/back-office-app-services/locals.tf

```diff
@@ -26,6 +26,7 @@ locals {
         APPLICATIONS_INSPECTOR_GROUP_ID          = var.azuread_applications_inspector_group_id
         NODE_ENV                                 = var.node_environment
         OS_PLACES_API_KEY                        = local.secret_refs["os-places-api-key"]
+        SESSION_SECRET                           = local.secret_refs["session-secret"]
```

Comment:
Make sure to add the referred secret name `session-secret` in the `secret_names` list on line 57 so that Terraform will generat a Key Vault secret to hold the `session-secret` value you're making reference to.